tools/install.py

```diff
@@ -260,6 +277,10 @@ def wanted_v8_headers(files_arg, dest):
       'deps/zlib/zlib.h',
     ], 'include/node/')
 
+  if sys.platform == 'zos':
+    zoslibinc = os.environ.get('ZOSLIB_INCLUDES')
```

Comment:
Do we need explicit logic to handle the case if ZOSLIB_INCLUDES is not set or set properly?

Reply:
yes and they are handled in #14 and https://github.com/ibmruntimes/node-gyp/pull/1

Reply:
I believe handle that in our internal build tools, should we add a check if we want to upstream it? @gabylb @IgorTodorovskiIBM

Reply:
Yeah this envar is set in our `setupenv` which is sourced by the build script that ends up running this `install.py`. But in case user ran `install.py` outside of that build script, would be better checking at least if `ZOSLIB_INCLUDES` envar is set.